### ISSUE TITLE
Date in new project page

### ISSUE THREAD

**Issue by mypalal84** [cleex1004/portfolio#16]
Instead of making the publisher enter a date on the new project, it would be nice if you could use code to automatically do that for them, along with a published checkbox. That page looks really good though.

**Comment by cleex1004** [comment]
yes, but what if i'm entering stuff for an old project and i dont want todays date on it?
